### PR TITLE
Added the possibility to get colors

### PR DESCRIPTION
**Pull request by oliverschwendener**
Title says it all ;)

### CODE REVIEW

File: src/AppBridgeNative.ts [hunk 1]
```diff
@@ -13,6 +21,26 @@ const assets: AppBridgeAssets = {
     },
 };
 
+const colors: AppBridgeColors = {
+    async getColorsByIds(colorIds: number[]): Promise<Color[]> {
+        const response = await window.application.config.data.get<{ success: boolean; palettes: ColorPalette[] }>(
+            `/api/color/library/${window.application.config.context.project.id}`,
+        );
```
Comment (SamuelAlev) [inline src/AppBridgeNative.ts:28]: I would do it using `fetch` like [here](https://github.com/Frontify/app-bridge/blob/196310b84baba7acc9e7e63dce9fc96ee847d432/src/AppBridgeNative.ts#L55)

File: src/AppBridgeNative.ts [hunk 2]
```diff
@@ -13,6 +21,26 @@ const assets: AppBridgeAssets = {
...
+
+        return response.palettes
+            .reduce((previous, current): Color[] => previous.concat(current.colors), <Color[]>[])
+            .filter((color) => colorIds.includes(Number(color.id)));
+    },
+
+    async getAvailableColors(): Promise<Color[]> {
+        const response = await window.application.config.data.get<{ success: boolean; palettes: ColorPalette[] }>(
+            `/api/color/library/${window.application.config.context.project.id}`,
+        );
```
Comment (SamuelAlev) [inline src/AppBridgeNative.ts:38]: Same here

File: src/AppBridgeNative.ts [hunk 3]
```diff
@@ -13,6 +21,26 @@ const assets: AppBridgeAssets = {
...
+
+        return response.palettes.reduce((previous, current): Color[] => previous.concat(current.colors), <Color[]>[]);
```
Comment (SamuelAlev) [inline src/AppBridgeNative.ts:40]:
```suggestion
        return response.palettes.map((palette) => palette.colors).flat(1);
```

File: src/window.d.ts [hunk 4]
```diff
@@ -10,6 +10,9 @@ export declare global {
                         id: number;
                     };
                 };
+                data: {
+                    get: <T>(url: string) => Promise<T>;
+                };
```
Comment (SamuelAlev) [inline src/window.d.ts:15]: If we use `fetch` this is then not needed

**Review by SamuelAlev — CHANGES_REQUESTED** [review]
Feature wise works fine but maybe a point worth to invest in

File: src/AppBridgeNative.ts (hunk 1 above)
Comment (oliverschwendener) [reply src/AppBridgeNative.ts]: Done.

File: src/AppBridgeNative.ts (hunk 3 above)
Comment (oliverschwendener) [reply src/AppBridgeNative.ts]: This is apparently an ES2019 feature, so I'll leave it as it is. We can refactor this as soon as we add "ES2019" to the "lib" option in `tsconfig.json`.

File: src/window.d.ts (hunk 4 above)
Comment (oliverschwendener) [reply src/window.d.ts]: Removed.

**Comment by oliverschwendener** [comment]
As I have adressed all the important things you pointed out in the code review, I'll merge this now.

**Pull request merged**